code/src/components/LoadingLottie.js

```diff
@@ -0,0 +1,20 @@
+// loading status
+import React from 'react'
+import Lottie from 'lottie-react-web'
+import animation from 'animation/burger.json'
+
+
+export const LoadingLottie = () => {
+  return (
+    <>
```

Comment:
No need for a fragment here. React components can only return a single child (or element), so fragments are useful when you want to return multiple children, but don't want to wrap them in an unnecessary div or other html element. In this case, you're only returning a single child (a `<Lottie />` component), so no fragments needed